cmd/extract/extract.go

```diff
@@ -30,7 +30,7 @@ import (
 func main() {
 	paths := []string{"cmd", "pkg"}
 	functions := []string{"translate.T"}
-	output := "pkg/minikube/translate/translations"
+	output := "translations"
```

Comment:
at first look it wasn't clear to me output was a filename, I suggest a better name, may be `f` or `outFile` ?

Reply:
`output` is a directory

Reply:
then maybe `outDir`